src/main/java/uk/gov/hmcts/reform/wataskmonitor/domain/jobs/adhoc/createtasks/ElasticSearchRetrieverParameter.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.wataskmonitor.domain.jobs.adhoc.createtasks;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import uk.gov.hmcts.reform.wataskmonitor.domain.jobs.RetrieveCaseListParam;
+
+@ToString
+@EqualsAndHashCode(callSuper = false)
+@Getter
+public class ElasticSearchRetrieverParameter extends RetrieveCaseListParam {
+
+    private final String serviceAuthentication;
+
+    public ElasticSearchRetrieverParameter(String serviceAuthentication) {
+        super();
+        this.serviceAuthentication = serviceAuthentication;
+    }
+}
```

Comment:
When you have to implement the interface, It's nicer not to be forced to implement the method with the same parameters. Therefore, this class shows an example to use different params for different implementations. Yes, it might look redundant now, but I use it to set an example of how to implement the RetrieverService. 
If it becomes useless going further, I'm more than happy to clean up, and I just wanted to put some pattern to make it a bit more flexible.

Reply:
done